README.md

```diff
@@ -19,17 +19,17 @@ The goals of uni are very simple:
 uni provides 2 React components:
 
 1. for the server, just pass it some initial data and routes, and render the thing
-2. for the client, just pass it the data your rendered on the serve and the same routes you passed to the server component, and hydrate the thing
+2. for the client, just pass it the data your rendered on the server and the same routes you passed to the server component, and hydrate the thing
```

Comment:
Can you change `just pass it the data your rendered` to `just pass it the data you rendered`